packages/vuetify/src/components/VField/VField.sass

```diff
@@ -169,9 +172,8 @@
 .v-field__prepend-inner,
 .v-field__append-inner,
 .v-field__clearable
-  align-items: flex-start
+  align-items: center
```

Comment:
Now using `align-items: center` to vertically center align icons.

This was used to be done via `padding-top` which is not the best option for this case.

On prod, when using chips, icons center alignment is broken as well



![field-icon-v3](https://user-images.githubusercontent.com/5698884/235388198-9f0ab187-2458-40b7-8441-1bb54e446420.gif)